src/components/Tailwind/Cards/StakeCard.tsx

```diff
@@ -3,15 +3,17 @@ import React from 'react'
 interface StakeCardProps {
   title: string
   value: string
+  mainnet?: boolean
```

Comment:
Suggestion: instead of the parent component passing this everytime, would it be more convenient to place the chainId check inside this component instead?

Reply:
@schystz if I put the chainId check inside the component every card will be having the Mainnet indicator instead of just the TVL card

Reply:
Oh I see @geonellov . In that case, can we just rename `mainnet` to `displayMainnetIndicator` just to be super clear on the intention?

Reply:
@schystz done